### ISSUE TITLE
Can't get print mode: custom to work with custom buttons

### ISSUE THREAD

**Issue by JIsbak** [Igor-Vladyka/leaflet.browser.print#86]
**Describe the bug**
When using the custom print button functionality I can't get the print mode: custom to work. It automatically print without letting me select an area. When I use the build in controls, everything is working as it should.

**To Reproduce**
```
L.control.browserPrint({
            position: 'bottomleft',
            closePopupsOnPrint: false,
            printModes: ["Portrait", "Landscape", "Auto", "Custom"],
            manualMode: false
        }).addTo(this.map);
```

And then the following in the custom button:
```
var modeToUse = L.control.browserPrint.mode.custom();
this.map.printControl.print(modeToUse);
```

**Expected behavior**
Should work as when using the build in controls.

**Desktop (please complete the following information):**
 - MAC OS
 - Chrome Version 81.0.4044.129 (Official Build) (64-bit)

**Comment by Igor-Vladyka** [comment]
Hello,

Thanks for noticing it. It was fixed in last version 1.0.5. Please take a look and let me know if any.

Regards,
Igor

**Comment by JIsbak** [comment]
Hi Igor,

I works👍 Thank you for the quick feedback!

Cheers,
Jens
